sirepo/pkcli/db.py

```diff
@@ -34,3 +34,106 @@ def _inc(m):
                 break
             with open(str(fn), 'w') as f:
                 f.write(t)
+
+
+def upgrade_runner_to_job_db(db_dir, sbatch_poll_secs):
```

Comment:
I don't understand why sbatch_poll_secs is needed. None of the jobs in the past could have used it.

Reply:
It's not needed. I just copy and pasted the NEXT_REQUEST_SECONDS dict and clearly didn't apply much thought. I'm using the poll_seconds() method now which kills two birds with one stone.